.github/workflows/build.yml

```diff
@@ -59,8 +66,10 @@ jobs:
           first_file_path: ./test.data
           second_file_path: integration/testdata/Expected_output.data
           expected_result: PASSED
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       - name: Dry-run release snapshot
         uses: goreleaser/goreleaser-action@v2
         with:
-          version: v0.169.0
+          version: v0.179.0
```

Comment:
somewhere between goreleaser 0.169.0 and 0.179.0 the support for the docker "use" option was added or changed, so updating this